oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -278,6 +277,17 @@ public GoogleCredentials createScoped(String... scopes) {
     return createScoped(ImmutableList.copyOf(scopes));
   }
 
+  /**
+   * If the credentials support automatic retries, creates a copy of the identity with the provided
+   * retry strategy
+   *
+   * @param defaultRetriesEnabled a flag enabling or disabling default retries
+   * @return GoogleCredentials with the new default retries configuration.
+   */
+  public GoogleCredentials createWithCustomRetryStrategy(boolean defaultRetriesEnabled) {
```

Comment:
This method name is awkward with the boolean value. I would expect to be providing a retry strategy as the parameter from reading the method name.

Reply:
It is not perfect, but still fits. I tried to come up with something better but did not :) If you have suggestion - please let me know. Also, this interface is part of proposed AIP: [go/aip4117](https://goto.google.com/aip4117)

Reply:
`enableRetries()` ??

Reply:
you mean createWithDefaultRetriesEnabled?

Reply:
kept the name as is